src/platform/Linux/ConfigurationManagerImpl.cpp

```diff
@@ -67,6 +67,9 @@ CHIP_ERROR ConfigurationManagerImpl::Init()
     err = Internal::GenericConfigurationManagerImpl<PosixConfig>::Init();
     SuccessOrExit(err);
 
+    // undo GenericDeviceInstanceInfoProvider being set as default DeviceInstanceInfoProvider
+    SetDeviceInstanceInfoProvider(&DeviceInstanceInfoProviderMgrImpl());
```

Comment:
This should be handled in InitChipStack(), no?

Reply:
This was fixed in #21259

Reply:
Ok, I removed the changes to src/platform/Linux/ConfigurationManagerImpl.cpp